### PR TITLE
Add enforcement steps and examples to the CoC.

### PR DESCRIPTION
**Pull request by indirect**
package.community has done a terrific job with their clear explanation
of what enforcing a CoC looks like. Bundler is a better place when
everyone knows what kind of behavior is expected, and what will happen
if those expectations are not met, for whatever reason.

### CODE REVIEW

**Comment by colby-swandale** [comment]
> Contact anyone in the Maintainers list on the Bundler website
Maybe we should have an email address (codeofconduct@bundler.io?) that can be used to contact the core team rather then pinging an individual member but contact an individual member if you're reporting a core team member?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,42 +1,136 @@
...
 
-Harassment includes offensive verbal comments related to level of experience, gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, ethnicity, age, religion, nationality, the use of sexualized language or imagery, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
+**The admin team will prioritize the well-being and comfort of the recipients of the violation over the comfort of the violator.** See [some examples below](#enforcement-examples).
 
-Individuals asked to stop any harassing behavior are expected to comply immediately.
+## Our Pledge
 
-Maintainers, including the core team, are also subject to the anti-harassment policy.
+In the interest of fostering an open and welcoming environment, we as members of the package.community community pledge to making participation in our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, technical preferences, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (segiddins) [inline CODE_OF_CONDUCT.md]: This says `package.community`, is that intentional?

**Comment by hsbt** [comment]
@indirect Thank you for mentioning me. I have no opinion about enforcement step related CoC violation.

I will focus to write a code and resolve the faced problem.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (indirect) [reply CODE_OF_CONDUCT.md]: No, blah. Thanks for catching that.

**Comment by indirect** [comment]
@colby-swandale that is a great idea, and I was somehow convinced that I had put writing an email to the entire team as one of the options. We have team@bundler.io set up right now. 👍

**Comment by indirect** [comment]
@hsbt thank you for your feedback. I wanted to make sure everyone in the active maintainers team got a chance to review this before it was merged. 👍

**Comment by indirect** [comment]
Thanks for the feedback, everyone!

@bundlerbot r+

**Pull request merged**